src/utils.jl

```diff
@@ -0,0 +1,49 @@
+StatsBase.describe(X::NullableVector) = StatsBase.describe(STDOUT, X)
+
+function StatsBase.describe(io::IO, X::NullableVector{<:Real})
+    if allnull(X)
+        println(io, " * All null * ") && return
```

Comment:
This will fail since `println` returns `Void` and it's being used in a boolean context. Having `return` on its own line will fix it.